misc/assets/inforequests.svg

```diff
@@ -0,0 +1,248 @@
+<svg version="1.1" viewBox="0 0 1000 1350" xmlns="http://www.w3.org/2000/svg">
+  <style>
+      text {
+        font-family: sans-serif;
+      }
+      .external{
+        stroke-dasharray: 3pt;
+      }
+      .caption {
+        font-size: 12pt;
+        text-anchor: middle;
+        dominant-baseline: central;
+      }
+      .note {
+        font-size: 9pt;
+        text-anchor: middle;
+        dominant-baseline: ideographic;
+      }
+      .left-rotation{
+        transform: rotate(-90deg);
+      }
+      .right-rotation{
+        transform: rotate(90deg);
+      }
+      .relation {
+        fill: none;
+        stroke: black;
+        stroke-width: 1pt;
+      }
+      .relation.from-many {
+        marker-start: url(#relation-many)
+      }
+      .relation.to-many {
+        marker-end: url(#relation-many)
+      }
+      .relation.from-optional {
+        marker-start: url(#relation-optional)
+      }
+      .relation.to-optional {
+        marker-end: url(#relation-optional)
+      }
+      .relation.to-link_attribute {
+        marker-end: url(#link_attribute)
```

Comment:
Preco miesas `-` s `_`? sak to kole oci. Vsade pouzivame `-` v nazvoch class a id, tak ich aj pouzivajme.

Reply:
:ok: